### DISCUSSION TITLE
I can't visit my website.

### DISCUSSION THREAD

**Discussion by yeo311** [vercel/community#6809] [Help] [answered]
### Summary

I can't access to my project's website, deployment is fine, logs show it responding with a 200OK.
However, when I access to the website in my browser, I can't access it.
The project name is "my-xellop"

### Example

https://my-xellop.vercel.app/

### Steps to Reproduce

Access to https://my-xellop.vercel.app/

**Comment by cokid1998** [comment, marked as answer]
https://github.com/orgs/vercel/discussions/6807

Maybe only Korea has a block of dns with vercel.app attached.

한국만 vercel.app이 붙어있는 도메인에 뭔가 문제가 있는거 같습니다.
